torch/testing/_internal/common_methods_invocations.py

```diff
@@ -678,6 +678,39 @@ def sample_inputs_index_select(op_info, device, dtype, requires_grad):
                         0, torch.tensor(0, dtype=torch.int64, device=device))),
             )
 
+# Missing to test the nondeterminism of the operation
+# https://github.com/pytorch/pytorch/issues/53352
+def sample_inputs_index_add(op_info, device, dtype, requires_grad):
+    # These testa are pretty much the same as those from index_copy.
```

Comment:
Merging sounds good

Reply:
`index_copy` has not been merged yet, so this will have to wait.

Reply:
This is one of the cases where stack of PRs would be convenient.